downstream/modules/platform/proc-operator-deploy-central-config.adoc

```diff
@@ -49,7 +49,7 @@ The following procedure simulates a scenario where you have {ControllerName} as
 . Click btn:[Create].
 . To access your new instance, see <<Accessing the platform gateway>>.
 
-NOTE: If you have an existing controller with a managed Postgres pod, after creating the *AnsibleAutomationPlatform* resource your {ControllerName} instance will continue to use that original Postgres pod. If you were to do a fresh install you would have a single Postgres managed pod for all instances. 
+NOTE: If you have an existing controller with a managed Postgres pod, after creating the *Ansible Automation Platform* resource your {ControllerName} instance will continue to use that original Postgres pod. If you were to do a fresh install you would have a single Postgres managed pod for all instances. 
```

Comment:
attribute and your {CotrollerName} instance continues to use

If you do a fresh install you have( is one created?) a single Postgres managed pod...